config/manager/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: system
```

Comment:
I'm not really sure how we ended up with a namesapce resource, but if we are updating all this I'd probably prefer we just put it back where it's usually found in the manager yaml see https://github.com/Kuadrant/kuadrant-operator/blob/main/config/manager/manager.yaml